CubeLineMomentManual.md

```diff
@@ -49,9 +49,10 @@ To run in ipython use:
 
 **spatial_mask_limit [float]:** Multiplier for noise-based spatial masking. Signal less than spatial_mask_limit times RMS noise is masked. Example: 2
 
+**sample_pixel [str, optional]:** A set of (x,y) coordinates to sample from the cutout cube to create diagnostic images.  Assumed to be in (RA,Dec) coordinates in a ds9 regions file, and must be within the cutout image area.  If left as `None`, no diagnostic images will be made.
```

Comment:
if it's in a region file, it can be in any coordinate system

Reply:
Correct.  Can be any coordinate system.  Will update docs.